sr_gui_controller_tuner/src/sr_gui_controller_tuner/controller_tuner.py

```diff
@@ -498,53 +537,56 @@ def refresh_controller_tree_(self, controller_type = "Motor Force"):
                         frame_buttons = QFrame()
                         layout_buttons = QHBoxLayout()
                         btn_plot = QPushButton("Plot")
-                        self.ctrl_widgets[ motor_name ]["btn_plot"] = btn_plot
-                        self.ctrl_widgets[ motor_name ]["btn_plot"].clicked.connect(partial(self.on_btn_plot_pressed_, motor_name, self.ctrl_widgets[ motor_name ]["btn_plot"]))
+                        self.ctrl_widgets[motor_name]["btn_plot"] = btn_plot
+                        self.ctrl_widgets[motor_name]["btn_plot"].clicked.connect(partial(self.on_btn_plot_pressed_,
+                                                                                  motor_name, self.ctrl_widgets[motor_name]["btn_plot"]))
                         layout_buttons.addWidget(btn_plot)
 
                         if self.controller_type in ["Position", "Muscle Position", "Mixed Position/Velocity"]:
                             #only adding Move button for position controllers
                             btn_move = QPushButton("Move")
-                            self.ctrl_widgets[ motor_name ]["btn_move"] = btn_move
-                            self.ctrl_widgets[ motor_name ]["btn_move"].clicked.connect(partial(self.on_btn_move_pressed_,motor_name, self.ctrl_widgets[ motor_name ]["btn_move"]))
+                            self.ctrl_widgets[motor_name]["btn_move"] = btn_move
+                            self.ctrl_widgets[motor_name]["btn_move"].clicked.connect(partial(self.on_btn_move_pressed_, motor_name,
+                                                                                      self.ctrl_widgets[motor_name]["btn_move"]))
                             layout_buttons.addWidget(btn_move)
                             frame_buttons.setLayout(layout_buttons)
 
-                        self._widget.tree_ctrl_settings.setItemWidget(  motor_item, 0, frame_buttons )
+                        self._widget.tree_ctrl_settings.setItemWidget(motor_item, 0, frame_buttons)
 
-                    for index_item,item in enumerate(ctrl_settings.headers):
+                    for index_item, item in enumerate(ctrl_settings.headers):
                         if item["type"] == "Bool":
                             check_box = QCheckBox()
 
-                            if parameter_values["sign"] == 1.0:
-                                check_box.setChecked(True)
-
-                            check_box.setToolTip("Check if you want a negative sign\n(if the motor is being driven\n the wrong way around).")
-                            self._widget.tree_ctrl_settings.setItemWidget(  motor_item, index_item, check_box )
+                            param_name = item["name"].lower()
+                            param_val = parameter_values[param_name]
+                            check_box.setChecked(param_val)
+                            if "sign" in param_name:
```

Comment:
I can now see the possible problems indeed. 
my example was for instance velocity/sign but then the tooltip might not be 100% relevant anyway. 
So, okay for equal for sign (param_name=="sign") as it was the original design, but of course only for the tooltip. 

Regarding the "in" in check_prefix I think this is more tricky. 
I did that due to possibly trailing slashes or underscores in the parameters (rh/ lh/ or rh_ lh_) we compare the prefix with. It also helped fetching both hands if no prefix was selected but hand_ids would have two values. empty always is "in" something so both the rh and lh matched and I could then warn that the empty prefix could not work due to the presence of two hands.

I can rework that to make the check_prefix more explicit but I would need to remove trailing underscores and slashes from the parameters I compare with prefix. 

Would you prefer that ?